README.md

```diff
@@ -207,6 +219,14 @@ class Welcome_test extends TestCase
 
 See [How to Write Tests](https://github.com/kenjis/ci-phpunit-test/blob/master/docs/HowToWriteTests.md) for details.
 
+## How to generate test result report
+
```

Comment:
@biechao Add instructions to install Phing. `composer require phing/phine --dev`